servers/mu/src/routes/middleware/withMetrics.js

```diff
@@ -0,0 +1,47 @@
+import { always } from 'ramda'
+import { histogramWith } from '../../domain/clients/metrics'
```

Comment:
Since the MU is type `module` this import requires an extension ie `/metrics.js`. I doubt the MU will start without throwing an error.

Also, a small convention is to add a space between it and dependencies resolved from `node_modules`. Not required, just helps keeps thing orderly.